src/DotNetProjectFile.Analyzers/Rule.cs

```diff
@@ -144,6 +144,16 @@ public static class Rule
         severity: DiagnosticSeverity.Warning,
         isEnabled: true);
 
+    public static DiagnosticDescriptor IncludePackageReferencesOnce => New(
+        id: 0013,
+        title: "Include package references once",
```

Comment:
I'd say `only once` to be more clear about the actual problem

Reply:
I fully agree!